.gitignore

```diff
@@ -5,3 +5,4 @@ dist/
 public_html/
 flow-coverage
 coverage
+.vscode/
```

Comment:
I think this file should probably only have project-specific ignores, while editor specific ignores can go into each person's personal global gitignore. Here are some steps I found on adding a global gitignore:

https://gist.github.com/subfuzion/db7f57fff2fb6998a16c